static/styles/bootstrap/bootstrap.css

```diff
@@ -3126,6 +3126,20 @@ table th[class*="span"],
   border-color: #ddd;
   border-color: rgba(0, 0, 0, 0.15);
 }
+.well-l {
+
+  min-height: 20px;
+  padding: 24px;
+  margin-bottom: 20px;
+  background-color: #f5f5f5;
+  border: 1px solid #e3e3e3;
+  -webkit-border-radius: 4px;
+     -moz-border-radius: 4px;
+          border-radius: 4px;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.05);
```

Comment:
@samsruti these features are new, css3 stuff. They are unsupported in !any browsers. Eden has to run properly on as many browsers as possible. Better to add prefixes. (I was going to recommend prefixfree by Lea verou but ssf doesn't need a whole new lib just for 2-3 css3 rules).